### PR TITLE
feat: slider, grid, typography added.

### PR DESCRIPTION
**Pull request by krishnakant01**
Hey, I've added a slider, grid, and typography. Please review it.

### CODE REVIEW

File: components/grid.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                midfielder...</p>
+            <button class="btn btn-text-primary card-btn">READ MORE</button>
+        </div>
+    </div>
+
+</div>
+
+<p class="sub-header">Example of 4 column grid layout</p>
+
+<p>Add <span class="nav-U-I red-text">grid-4-column-layout </span> class on wrapper div which is wrapping 2 divs.</p>
```
Comment (tusharkandpal) [inline components/grid.js]: Can change this class name `nav-U-I`

File: docs.html [hunk 2]
```diff
@@ -52,6 +52,9 @@ <h2 class="sidebar-header">Components</h2>
                 <a href="#Navigation">
                     <li class="sidebar-link-item">Navigation</li>
                 </a>
+                <a href="#Grid">
```
Comment (tusharkandpal) [inline docs.html]: These `Navigation` `Grid` `Slider` should be in lower case in my opinion

**Review by tusharkandpal — APPROVED** [review]
Great with the Slider part! Suggested few changes.

& You could have raised separate PRs for all these 3 parts. Will be easy to review

File: components/grid.js (hunk 1 above)
Comment (krishnakant01) [reply components/grid.js]: yes, actually I've used it at too many places to change now, do you have any idea where I can change them all at once?

Comment (tusharkandpal) [reply components/grid.js]: `Search & replace all` feature you can try in VS code

**Review by mariyasada — COMMENTED** [review]
It is difficult to review  42 files. trying to make separate branch for particular feature

**Comment by krishnakant01** [comment]
no actually it was 12 files initially, then I just refactored the code so it became 42, sorry for the inconvenience.

**Comment by krishnakant01** [comment]
> It is difficult to review 42 files. trying to make separate branch for particular feature

no, actually it was 12 files initially, then I just refactored the code so it became 42, sorry for the inconvenience.

**Pull request merged**